### DISCUSSION TITLE
PoE and NMEA OneNet

### DISCUSSION THREAD

**Discussion by ghost** [peterantypas/maiana#137] [Ideas]
With the development of NMEA OneNet, have you given any consideration to moving to standard PoE and OneNet for communication without an adapter (plugged directly into a PoE router)?

**Comment by peterantypas** [comment]
It depends on how ‘open’ it is. Regardless, an adapter will still be needed.Sent from my iPhoneOn Nov 30, 2023, at 3:36 AM, masonleschyna ***@***.***> wrote:﻿
With the development of NMEA OneNet, have you given any consideration to moving to standard PoE and OneNet for communication without an adapter (plugged directly into a PoE router)?

**Reply by ghost** [reply]
Why would an adapter be needed? I would envision a number of sensors connected to a PoE router via ethernet cabling. My understanding is that NMEA OneNet allows NMEA 2000 mesaages to be sent over IP. What do you mean how open?

**Reply by kagouraki** [reply]
Maiana has nothing to do with a lan network. It just uses ethernet cable because it is convenient. In order to be part of a lan it needs an ethernet chip. Then yes... it can be connected to a lan

**Reply by ghost** [reply]
Yes, I understand that. If LAN was incorporated into the base unit though, it would eliminate the need for a separate adapter module. It could then be used with and LAN connected device.

**Reply by kagouraki** [reply]
Yes. Thats a big if though... means new pcb and design and more cost


Peter will deside if this path is worth it. I dont think so. Because then you are limited and need definitely a hub or router

**Comment by peterantypas** [comment]
To incorporate an ethernet PHY into the main unit it will have to grow in
size and incorporate a second processor.
